app/renderer/lib/extensionsUtil.js

```diff
@@ -177,6 +184,8 @@ module.exports.getExtensionKey = (extensionId) => {
     case metamask:
       extensionSetting = settings.METAMASK_ENABLED
       break
+    case betterttv:
+      extensionSetting = settings.BETTERTTV_ENABLED
```

Comment:
this needs a break afterwards- linter is failing:
`/home/travis/build/brave/browser-laptop/app/renderer/lib/extensionsUtil.js:189:5: Expected a 'break' statement before 'default'.`